Source/Scene/Model.js

```diff
@@ -772,6 +781,10 @@ define([
         return JSON.parse(json);
     }
 
+    Model.getDefaultCacheKey = function(url) {
```

Comment:
Why put this on `Model` instead of just having a file-scoped function?  Do you plan to use this in the tests?